### ISSUE TITLE
Time is wrong when creating a new activity

### ISSUE THREAD

**Issue by dennisdenk** [kevinpapst/kimai2#783]
**Describe the bug**
When starting a new task the time is always in UTC, no matter what timezone you choose in the settings

**To Reproduce**
1. Create a new activity and look at the time

**Screenshots**
![grafik](https://user-images.githubusercontent.com/19978270/57587571-32dbd780-7507-11e9-98e6-f4286664ad2c.png)

**Additional context**
 - latest Version
 - Running in docker container

**Comment by kevinpapst** [comment]
Works on https://demo.kimai.org as expected, which btw runs with CEST as well.

When you change the user settings to Australia/Perth and click "create" again, the time is still shown in UTC? 

I would say this is an environment issue, but I have not enough docker experience to help.

Which docker image are you using?
Maybe this helps: https://github.com/tobybatch/kimai2/issues/6

**Comment by dennisdenk** [comment]
I just changed to Perth and the time is still in UTC.
The time in the docker container is synced with the host time.
I use the latest version by tobybatch
https://github.com/tobybatch/kimai2

**Comment by kevinpapst** [comment]
@tobybatch does timezone switching work on your end?

**Comment by tobybatch** [comment]
I'll check this and then follow it up on the container repo.

**Comment by amhambra** [comment]
Hi Team,
on the demo site from your website I try to create a new entry as tony_teamlead. I live in Berlin Timezone. My time was around 12:03 and the time the entry was created is 17:04 at the same date.

I checked as Admin that Timezone is set to Berlin.

Kindly advice,

I like the demo, but in case the "client browser" time and timezone is not taken for create this tool is not helpfull.

Cheers,
Frank

![20190525-timezone-berlin-time-1159-but-create-time-1659](https://user-images.githubusercontent.com/922436/58367935-b2917b00-7ee5-11e9-8858-6ac93a3355ed.jpg)
![20190525-timezone-berlin-time-1159-but-create-time-1659-02](https://user-images.githubusercontent.com/922436/58367936-b2917b00-7ee5-11e9-9575-36296c891971.jpg)
![20190525-timezone-berlin-time-1159-but-create-time-1659-03](https://user-images.githubusercontent.com/922436/58367937-b2917b00-7ee5-11e9-916b-6d3ca653f417.jpg)

**Comment by kevinpapst** [comment]
Hi Frank,
you are using a wrong configuration. You checked the default timezone for customers, which is just a form field when creating a new customer, its not used for anything else. This setting has nothing to do with the users time-records.

Each user has his own timezone settings, e.g. https://demo.kimai.org/en/profile/susan_super/prefs (think of a company with distributed teams across the globe).
This user timezone is the ONLY one that counts. Kimai does not take into account your browsers timezone. If you switch timezones, you have to define that in your preferences.

And thanks for our technical advices 👍 such input is very helpful. I could use of that, so please stick with Kimai :-) Kimai already stores the times in UTC and additionally the timezone they were recorded in. BUT it only stores one timezone per entry, the starting one. 

Your absolutely rare edge-case (I don't design an application for 0.1% of the users) starting one record in Helsinki and stopping it in Melbourne would still work. It would even be an error if I would use Helsinki to start and Melbourne time to stop the entry. 
I don't know how long a flight takes and if the timezone difference is realistic, but just for the sake of demonstration: assume you work the total flight time of 15 hours and approach Melbourne at the same date and time you left Helsinki. If I would use the locale timezones, then Kimai would record a zero duration.
So, you can try to change the timezone setting between starting and stopping a record, but it would always use the timezone when you started the record => which is in my view the only valid behaviour.

You could display the UTC time, but I will not add that to the core, as I don't think that anyone will use such a feature (who knows about UTC anyways besides "us developers"?).  But I don't mind a plugin which will do this.

Btw. next time, go ahead and open a new issue, this one is docker related.

**Comment by amhambra** [comment]
Hi Kevin, 

duhhh, 

now I see the User Timezone pref, switchted it to Berin and get the right starting Time! I think your technical description is valid, and I would say my "strechted timezone working" is a 0,01% Edge case. And storing the starting Timezone solves the problem suficiently. I try to setup Kimai in my Root server and will see how it goes for me.

Many Thanks for the healthy and fast response! And if I have other technical inside discussion points I let you know. Nice Weekend!

Cheers
Frank

**Comment by dennisdenk** [comment]
@kevinpapst because of your answer mentioning the user timezone settings I checked that at my installation. The standard user has UTC as timezone, so the behavior was totally normal. Sorry for not checking that in advance. Keep up your great work!

**Comment by kevinpapst** [comment]
Hm, I guess I should write a "first starter" page in the documentation, mentioning the most important settings & configurations which should be done first. 
Seems it is not as obvious as I hoped it would be.
Thanks for your feedback!
